test/Microsoft.AspNetCore.Antiforgery.Test/DefaultAntiforgeryTest.cs

```diff
@@ -381,7 +374,7 @@ public void GetAndStoreTokens_NoExistingCookieToken_Saved()
         }
 
         [Fact]
-        public void ValidateTokens_FromInvalidStrings_Throws()
+        public void TryValidateTokenSet_FromInvalidStrings_Throws()
```

Comment:
More odd naming since this method isn't what's throwing nor what's called in `// Act`.